client/components/checkbox/style.scss

```diff
@@ -0,0 +1,27 @@
+.form-checkbox {
+	margin-right: 8px;
+	cursor: pointer;
+	display: inline-flex;
+	align-items: center;
+	position: relative;
+
+	input {
+		margin: 0;
+
+		&:before {
+			display: none !important; // Prevent Calypso's Noticons font
+		}
+	}
+
+	.gridicon {
+		color: $blue-medium;
+		pointer-events: none;
+		position: absolute;
+		left: 1px;
+		top: 1px;
+	}
+
+	&.is-disabled .gridicon {
+		color: lighten( $gray, 10% );
```

Comment:
![checkbox-disabled](https://cloud.githubusercontent.com/assets/1715800/23485794/e768d96a-fed4-11e6-9406-d5d8377c34f9.png)

Works for me. Are you sure you put the right prop when testing? Should look like this (in the `package-list-item.js` file:
`<Checkbox checked={ selected } onChange={ onToggle } disabled />`

We aren't using it right now, but it's a use case so basic that I prefer to support it out-of-the-box.